pkg/kube/roles.go

```diff
@@ -139,7 +150,7 @@ func UpdateUserRoles(kubeClient kubernetes.Interface, jxClient versioned.Interfa
 		} else {
 			found := false
 			for _, subject := range envRole.Spec.Subjects {
-				if subject.Kind == userKind && subject.Name == userName {
+				if subject.Kind == userKind && subject.Name == userName && subject.Namespace == adminNs {
```

Comment:
we could have users in multiple namespaces. unlikely but possible - so lets at least check.